src/components/views/rooms/RoomList.js

```diff
@@ -341,7 +348,7 @@ module.exports = React.createClass({
                 const tagNames = Object.keys(room.tags);
 
                 // Apply TagPanel filtering, derived from FilterStore
-                if (!this.isRoomInSelectedTags(room, me, dmRoomMap)) {
+                if (!this.isRoomInSelectedTags(room)) {
```

Comment:
is `dmRoomMap` now redundant in this method?

Reply:
I don't think so, it's used to make the-- direct messages list.

Reply:
:+1: apparently this whole thing would be easier if I could read.